java-logger/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.streamzi.archetypes</groupId>
+    <artifactId>streamzi-archetypes-parent</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>streamzi-logger-archetype</artifactId>
```

Comment:
I think the artifactId would be better as streamzi-sink. Sink is a more general term and the -archetype is duplicated in the parameter passed to archetype:generate